uwblueprint/project-read-backend
# Enable browsable API in debug mode

### Notion ticket link



[Spring cleaning 🌼 ](https://www.notion.so/uwblueprintexecs/9cb454262ed84c6299c0682c23f92ec5?v=1076bd30711a4a01838becee05f8b5a3&p=4b8bb43e63664dceaf6a582dfe7d0806)



### Implementation description

- with our custom auth, we're no longer able to access the browsable API since DRF uses session auth (see [StackOverflow post](https://stackoverflow.com/a/38626166))
- if running in debug mode, enable session auth so that we can use the browsable API while building things



### Steps to test

1. `python manage.py test`
2. go to https://localhost:8000/families, sign in with your local user, and verify data shows up



### What should reviewers focus on?

- @danielchen115 do you think it's safe to enable session auth when debug=false?

### Checklist
